src/attention/utils/helpers.ts

```diff
@@ -163,20 +164,31 @@ export async function useRecompute(state: AttentionState) {
     middleware: [
       offset({ mainAxis: state?.distance ?? 8, crossAxis: state?.skidding ?? 0}), // offers flexibility over how to place the attentionEl towards its targetEl both on the x and y axis (horizontally and vertically).
       state?.flip && flip({ //when flip is set to true it will move the attentionEl's placement to its opposite side or to the preferred placements if fallbackPlacements has a value
-        fallbackAxisSideDirection: 'start', // the preferred placement axis fit when flip is set to true and fallbackPlacements does not have a value. 'start' represents 'top' or 'left'.
+        crossAxis: state?.crossAxis, //checks overflow to trigger a flip. When disabled, it will ignore overflow
         fallbackPlacements: state?.fallbackPlacements,
       }),
-      !state?.flip && autoPlacement(), // when flip is set to false, it will instead call autoPlacement() that will choose the placement that has the most space available automatically. You can only use either flip() or autoPlacement(), and not combined. 
-      shift({ padding: 16}),
       !state?.noArrow && state?.arrowEl && arrow({ element: state?.arrowEl }),
+      hide({ //will hide the attentionEl when it appears detached from the targetEl 
+        strategy: 'escaped', //default strategy is 'referenceHidden'
+      }),
+      hide(),
```

Comment:
Should the `hide()` be added twice here in the middleware array?

Reply:
I read in [floating-ui's](https://floating-ui.com/docs/hide#strategy) documentation that if you wanted to use several hide-strategies you can just call hide() multiple times in the middleware array. I have now made attentionEl behave the same way as in floating-ui's [example](https://floating-ui.com/docs/hide) where both `escaped` and `referenceHidden` is being used. :)

Reply:
But maybe I should add another comment for why hide() is being added twice? 🤔

Reply:
Updated in this commit: [b395f6e](https://github.com/warp-ds/core/pull/16/commits/b395f6edf0c865d31279b1c68a2cd2674d58e271)

Reply:
Decided in the end to remove the use of `escaped` hide-strategy since that would be more of a design question, whether we would want the `attentionEl` to get `opacity: 0.5` when `targetEl` is being clipped. So now the `attentionEl` will only hide itself when `targetEl` is no longer in viewport. Updated in this commit: [e14568e](https://github.com/warp-ds/core/pull/16/commits/e14568e914e02d825eca0e262e391411e7bb5c5b)